tests/tests.toml

```diff
@@ -346,6 +346,12 @@ filename = "../integration_tests/print_01.py"
 c = true
 cpp = true
 
+[[test]]
+filename = "../integration_tests/test_str_comparison.py"
+llvm = true
+c = true
+cpp = true 
+
```

Comment:
```suggestion
```
These are not needed. Just testing with `cpython` and `llvm` should be enough, I think.

Reply:
I deleted it.